discourse_rock/patches/rake.patch

```diff
@@ -0,0 +1,87 @@
+--- a/lib/tasks/discourse-charm.rake
++++ b/lib/tasks/discourse-charm.rake
```

Comment:
I would change the name of the file from `rake.patch` to something more explicit, like `discourse-charm.rake` or `user-tasks.rake`